### PR TITLE
Segregation of Header and Footer

### PR DESCRIPTION
**Pull request by sectrait**
Followed the guidance from [StackOverflow Question: common-header-footer-with-static-html](https://stackoverflow.com/questions/418503/common-header-footer-with-static-html) and created a pull request.

### CODE REVIEW

**Comment by paramvir1994** [comment]
Hey @sectrait,

This looks good to me but probably couple of concern should be addressed before merging to master as follows:

1. Your pushed branch is behind master that creates a conflict, hence cannot be merged. Therefore, you might have to pull the latest changes first and then commit back changes again.
2. We should keep the project structure generic as it would be easier for anyone to jump in and start working, and make it easy to locate files. So here are my suggestions to arrange the files:
 ```
root/
├── assets/
│   └── img/
|── pages/
|    ├──header.html
|    └── footer.html
|── script/
|    └── main.js
|── styles/
|    └── main.css
└── index.html
```
3. The last thing I'd strongly suggest is to pull out the script section from `index.html` and put it in `script/main.js` and then use `<script type="text/javascript" src="main.js"></script>`

I hope this is okay, or else we can discuss this during today's call.

File: assets/header_footer/footer.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<footer class="text-gray-700 body-font">
```
Comment (paramvir1994) [inline assets/header_footer/footer.html]: Move file to `pages/footer.html`

File: assets/header_footer/header.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<header class="text-gray-700 body-font">
```
Comment (paramvir1994) [inline assets/header_footer/header.html]: move file to `pages/header.html`

File: index.html [hunk 3]
```diff
@@ -1,92 +1,49 @@
...
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>Khalsa Serving</title>
   <meta name="description" content="Khalsa Serving">
   <meta name="author" content="ServingKhalsa">
   <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script>
```
Comment (paramvir1994) [inline index.html]: Remove script out of `index.html` and include it in `script/main.js` file.

File: assets/header_footer/header.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    <div class="container mx-auto flex flex-wrap p-5 flex-col md:flex-row items-center">
+        <a class="flex title-font font-medium items-center text-gray-900 mb-4 md:mb-0">
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" stroke="currentColor" stroke-linecap="round"
+                stroke-linejoin="round" stroke-width="2" class="w-10 h-10 text-white p-2 bg-orange-500 rounded-full"
+                viewBox="0 0 24 24">
+                <path d="M12 2L2 7l10 5 10-5-10-5zM2 17l10 5 10-5M2 12l10 5 10-5"></path>
+            </svg>
+            <span class="ml-3 text-xl">Khalsa Serving</span>
+        </a>
+        <nav class="md:ml-auto flex flex-wrap items-center text-base justify-center">
```
Comment (paramvir1994) [inline assets/header_footer/header.html]: Please note that the header code has been updated in the current Master, so please update it too.

**Review by paramvir1994 — CHANGES_REQUESTED** [review]

**Comment by sectrait** [comment]
Perfect @paramvir1994 . Working on it.

File: assets/header_footer/header.html (hunk 4 above)
Comment (sectrait) [reply assets/header_footer/header.html]: Done

File: assets/header_footer/footer.html (hunk 1 above)
Comment (sectrait) [reply assets/header_footer/footer.html]: Done

File: assets/header_footer/header.html (hunk 2 above)
Comment (sectrait) [reply assets/header_footer/header.html]: Done.

**Pull request merged**